src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Connection.cs

```diff
@@ -316,7 +324,16 @@ void IConnectionControl.Pause()
         void IConnectionControl.Resume()
         {
             Log.ConnectionResume(ConnectionId);
-            _socket.ReadStart(_allocCallback, _readCallback, this);
+            try
+            {
+                _socket.ReadStart(_allocCallback, _readCallback, this);
```

Comment:
Even though libuv uses a single thread, I assume ReadStart() calls into some OS function which itself can fail at any time.  So we need to either try/catch or check a status code.  Status code might have better perf but .NET convention for I/O is exceptions, and this particular exception will not be thrown often enough where it should impact real-world perf.

I think the current code is fine, and adding more code is just increasing complexity with no real benefit.